### DISCUSSION TITLE
[BUG] Application Freezes When Double-Clicking Title Bar

### DISCUSSION THREAD

**Discussion by bingling00** [spacedriveapp/spacedrive#2510] [General]
Version 0.2.14 ,macOS

When I double-click the title bar of the Spacedrive application window, the software unexpectedly freezes. This action seems to trigger an unintended behavior where the application becomes unresponsive, requiring a forced restart to regain functionality. There are no error messages displayed during this freeze, and the user interface simply ceases to react to any input.

**Comment by utkubakir** [comment]
This has been fixed in version 0.2.15, and we will be releasing it soon.
